pkg/globalconfig/values.go

```diff
@@ -23,6 +23,12 @@ var ValidOmitContainers = map[string]bool{
 	DBAContainer:          true,
 }
 
+// ValidRouterTypes is the list of valid router types
```

Comment:
imho belongs to a dedicated types package

If you use a map for this (which will make you troubles if you try to create a list of valid types btw) you should use `map[string]struct{}` which has the lowest impact on Golang.

Please also have a look at my Mutagen patch how I have solved it there.